README.md

```diff
@@ -121,3 +121,22 @@ mvn release:prepare release:clean -Darguments=-DskipTests
 ```
 
 With commands you are able to define a release, a tag and a next development version name in interactive way.
+
+## Issues found by the testsuite
+Any issue found in WildFly/EAP release will be reported into the JBoss issue tracker: https://issues.jboss.org/ . There are the following Jira projects:
```

Comment:
`issues.jboss.org` domain is not in use anymore. The current one is `issues.redhat.com` and it's named "Red Hat issue tracker".